default/python/AI/PriorityAI.py

```diff
@@ -519,15 +519,15 @@ def _calculate_military_priority():
                 status.get("fleetThreat", 0) ** 0.5
                 + 0.8 * status.get("max_neighbor_threat", 0) ** 0.5
                 + 0.2 * status.get("neighborThreat", 0) ** 0.5
-                + monster_threat ** 0.5
+                + monster_threat**0.5
```

Comment:
Seems unrelated to the title...?

Reply:
The new version of code formatter (black) works in a slightly different way.